cloud/src/test/scala/org/apache/spark/cloud/AzureInstantiationSuite.scala

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I am still not sure these tests are adding much. What error would it catch -- how would it catch a Jackson issue?